dev-3/dev-3/EntryPoint.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace dev_3
+{
+    /// <summary>
+    /// Program's entry point
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try//Try your string here
+            {
+                var yourString = "";
+                Transliterator c = new Transliterator(yourString);
+                Console.WriteLine(c.Translit());
+            }
+            catch (InvalidFormatException e)
+            {
+                Console.WriteLine(e.Message);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
```

Comment:
Better to use one Transliterator  object and set different sentences via property for processing